src/ports/analyticsDayData/utils.ts

```diff
@@ -77,3 +90,58 @@ export function getTimestampFromTimeframe(timeframe: AnalyticsTimeframe) {
       return 0
   }
 }
+
+// Rentals
+
+export function getRentalsAnalyticsDayDataQuery(
+  filters: AnalyticsDayDataFilters
+) {
+  const { from } = filters
+
+  const where: string[] = []
+
+  if (from) {
+    where.push(`date_gt: ${Math.round(from / 1000)}`)
+  }
+
+  return `
+    query AnalyticsDayData {
+      analytics: analyticsDayDatas(where: { ${where.join('\n')} }) {
+        id
+        date
+        rentals
+        volume
+        lessorEarnings
+        feeCollectorEarnings
+      }
+    }
+  `
+}
```

Comment:
What if we print the where section of the query blank if no `from` filter is performed?
```suggestion
  return `
    query AnalyticsDayData {
      analytics: analyticsDayDatas(${where.length > 0 ? `where: { ${where.join('\n')} }`: ''}) {
        id
        date
        rentals
        volume
        lessorEarnings
        feeCollectorEarnings
      }
    }
  `
}
```
This would make you have only one function for the query.

Reply:
Can we add tests to this new function?